### DISCUSSION TITLE
tesseract_ros in OSRF package repo?

### DISCUSSION THREAD

**Discussion by katettudelft** [tesseract-robotics/tesseract_ros#182] [Q&A] [answered]
Hi Levi.

I see on http://repositories.ros.org/status_page/ros_noetic_default.html?q=tesser that a lot of tesseract packages are available on the ros repos, but I notice that tesseract_ros is missing.

I'm curious if this is for any particular reason. :) Are there any plans to make it available this way?

Thanks!

**Comment by Levi-Armstrong** [comment, marked as answer]
Not really,  I am working on wrapping up a large refactor adding tesseract_task_composer then I was going to do a 0.14 release for tesseract, tesseract_planning, tesseract_qt and tesseract_ros on the build farm. There is one outstanding issue where for some reason when building from source with the binary installed it is finding the binaries first instead of the source packages in the workspace.

**Reply by katettudelft** [reply]
Thanks for the reply! That's great news - I look forward to trying out the release when it's done. :)

**Comment by gavanderhoorn** [comment]
Hi @Levi-Armstrong. Friendly ping :) Would you happen to have any time to look into releasing Tesseract?

**Reply by Levi-Armstrong** [reply]
I think we are very close to another release. I think after this [PR](https://github.com/tesseract-robotics/tesseract_qt/pull/22) is merged I will be able to move forward with another release.

**Reply by gavanderhoorn** [reply]
Just making sure: by *release* you mean Bloom-ing and having the ROS buildfarm build packages, right?

So the packages you mention in your previous comment: "tesseract_planning, tesseract_qt and tesseract_ros"?

**Reply by Levi-Armstrong** [reply]
Yea, assuming the dependencies are available to release trajopt.

**Reply by aashish-tud** [reply]
A friendly follow up. I see the [PR](https://github.com/tesseract-robotics/tesseract_qt/pull/22) you mentioned above is closed. I still cannot find `tesseract_ros` [here](http://repositories.ros.org/status_page/ros_noetic_default.html?q=tesseract_ros). Are there still plans to make the packages available?
